lib/manager/pipenv/artifacts.ts

```diff
@@ -28,14 +54,20 @@ export async function updateArtifacts({
     const localPipfileFileName = join(config.localDir, pipfileName);
     await outputFile(localPipfileFileName, newPipfileContent);
     const localLockFileName = join(config.localDir, lockFileName);
-    const cmd = 'pipenv lock';
+    const cmd = ['pip install pipenv', 'pipenv --user lock'];
```

Comment:
Won't this install pipenv even for users who are running with binarySource=auto?